### PR TITLE
Chris 3 sections

### PR DESCRIPTION
**Pull request by MinceMan**

### CODE REVIEW

File: library/src/main/java/com/twotoasters/sectioncursoradapter/adapter/datahandler/ArrayDataHandler.java [hunk 1]
```diff
@@ -110,6 +114,17 @@ public boolean hasStableIds() {
...
     }
 
+    @Override
+    public void registerObservable(DataChangeListener listener) {
+        mChangeListeners.add(listener);
+    }
+
+    @Override
+    public void unregisterObservable(DataChangeListener listener) {
+        mChangeListeners.add(listener);
```
Comment (patjackson52) [inline library/src/main/java/com/twotoasters/sectioncursoradapter/adapter/datahandler/ArrayDataHandler.java]: add should be remove

**Comment by patjackson52** [comment]
done with comments

File: library/src/main/java/com/twotoasters/sectioncursoradapter/adapter/datahandler/ArrayDataHandler.java (hunk 1 above)
Comment (MinceMan) [inline library/src/main/java/com/twotoasters/sectioncursoradapter/adapter/datahandler/ArrayDataHandler.java]: Good eye. Addressed comments.

**Comment by patjackson52** [comment]
+1

**Pull request merged**